projects/research/README.MD

```
@@ -2,6 +2,49 @@
+sudo dnf install fontconfig-devel clang-devel
+```
+
+### macOS
+
+```bash
+brew install fontconfig llvm
+```
+
+### Windows
+
+- Install LLVM from https://releases.llvm.org/ or via `winget install LLVM.LLVM`
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)